src/benchmark.cc

```diff
@@ -699,7 +743,11 @@ void RunBenchmark(const benchmark::internal::Benchmark::Instance& b,
         report.report_label = label;
         // Report the total iterations across all threads.
         report.iterations = static_cast<int64_t>(iters) * b.threads;
-        report.real_accumulated_time = real_accumulated_time;
+        if (b.use_manual_time) {
```

Comment:
I like to minimise surprises, and i think that if someone calls `->UseRealTime()->UseManualTime()` on a benchmark it should do the right thing (show real time and manual time).

yes, it might be cluttered in console mode but it's already fairly heavy-weight and the user has requested the extra data.